WultraMobileTokenSDK/Operations/Model/UserOperation/Screens/WMTPreApprovalScreen.swift

```diff
@@ -0,0 +1,69 @@
+//
+// Copyright 2023 Wultra s.r.o.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions
+// and limitations under the License.
+//
+
+import Foundation
+
+/// WMTPreApprovalScreen is the base class for Pre Approval screen classes
+///
+/// `type` define different kind of data which can be passed with operation
+/// and shall be displayed before operation is confirmed
+public class WMTPreApprovalScreen: Codable {
+    
+    /// type of PreApprovalScrren is presented with different classes (Starting with `WMTPreApprovalScreen*`)
+    public let type: PreApprovalScreenType
+    
+    public enum PreApprovalScreenType: String, Codable {
+        case info = "INFO"
+        case warning = "WARNING"
+        case qr = "QR"
```

Comment:
Was this discussed with backend devs it will be `QR`?

Reply:
Because we don't use it in decode, it's kinda unnecessary as of now.

Reply:
So it's not just QR on backend, see https://github.com/wultra/enrollment-server/pull/744#pullrequestreview-1403932536

Reply:
Yes. Set to "QR_SCAN"